wger/gym/templates/gym/partial_user_list.html

```diff
@@ -47,6 +49,44 @@
             -/-
         {% endif %}
     </td>
+    <td>
+        {#This is used to show the active status of the current member #}
+        {% if current_user.obj.get_full_name == 'Administrator'%}
+            {{ '-/-'}}
+        {% else %}
+            {% if current_user.obj.is_active%}
+                {{ 'Active' }} 
+            {% else %}
+                {{ 'Inactive' }} 
+            {% endif %}
+            
+        {% endif %}       
+    </td>
+    <td>
+        {% if current_user.obj.get_full_name == 'Administrator'%}
```

Comment:
This condition should be updated to make use of the `is_admin` method added to the user profile. Using the `full_name` isn't be a true check of the user's role.